src/d3a/setup/area_events/pv_event.py

```diff
@@ -32,7 +32,7 @@ def get_setup(config):
                 children=[
                     Area('H2 PV', strategy=PVStrategy(1, 80),
                          appliance=PVAppliance(),
-                         event_list=[StrategyEvents(12, {'panel_count': 10,
+                         event_list=[StrategyEvents(12, {'panel_count': 5,
```

Comment:
Sorry, can not see from the diff, why this change was necessary. Can you help me please? Thanks!

Reply:
It was not that necessary. The reason for this change was to expect a value multiplied by *10 and not by *20 after 12:00 during the test. That way it was easier for me to see that the traded energy after 12:00 was related to the traded energy before 12:00